scripts/known_good.json

```diff
@@ -6,7 +6,7 @@
             "sub_dir": "Vulkan-Headers",
             "build_dir": "Vulkan-Headers/build",
             "install_dir": "Vulkan-Headers/build/install",
-            "commit": "v1.3.275"
+            "commit": "v1.3.276"
```

Comment:
I usually update Vulkan-Utility-Libraries and Vulkan-Loader header version as well

Reply:
I'll wait for the repository to be updated and update both Vulkan-Utility-Libraries and Vulkan-Loader